### PR TITLE
Fixes #22 - update to phoenix 0.2.6

### LINKED ISSUE #22: Upgrade to Phoenix 0.2.6
See changelog: https://github.com/owncloud/phoenix/pull/2825

Please note that there was no stable release since a while, so expect that some issues might happen after upgrading.

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -178,4 +178,4 @@ watch:
 # protobuf:  $(GOPATH)/bin/protoc-gen-go $(GOPATH)/bin/protoc-gen-micro $(GOPATH)/bin/protoc-gen-microweb $(GOPATH)/bin/protoc-gen-swagger pkg/proto/v0/example.pb.go pkg/proto/v0/example.pb.micro.go pkg/proto/v0/example.pb.web.go pkg/proto/v0/example.swagger.json
 
 assets:
-	mkdir assets/ && curl -slL -o- https://github.com/owncloud/phoenix/releases/download/0.2.5/phoenix.tar.gz | tar xvzf - -C assets/
+	mkdir assets/ && curl -slL -o- https://github.com/owncloud/phoenix/releases/download/v0.2.6/phoenix.tar.gz | tar xvzf - -C assets/
```
Comment (DeepDiver1975) [inline Makefile:181]: tag name with v will disappear in the next version

File: pkg/assets/embed.yml [hunk 2]
```diff
@@ -10,8 +10,8 @@ compression:
 
 custom:
   - files:
-    - "../../assets/"
-    base: "../../assets/"
+    - "../../assets/phoenix/dist/"
```
Comment (DeepDiver1975) [inline pkg/assets/embed.yml]: package structure got messed up - will be fixed in next phoenix release

File: pkg/assets/embed.yml [hunk 3]
```diff
@@ -10,8 +10,8 @@ compression:
 
 custom:
   - files:
-    - "../../assets/"
-    base: "../../assets/"
+    - "../../assets/phoenix/"
```
Comment (tboerger) [inline pkg/assets/embed.yml]: Please strip the phoenix folder within the tar command within the makefile. Some consistent release building would be helpful.

File: Makefile [hunk 4]
```diff
@@ -57,7 +57,7 @@ sync:
 .PHONY: clean
 clean:
 	go clean -i ./...
-	rm -rf $(BIN) $(DIST)
+	rm -rf $(BIN) $(DIST) assets
```
Comment (butonic) [inline Makefile:60]: not sure about this. but since using a symllink to phoenix/dist folder does not work anyway we can get rid of the assets as well ...

Comment (DeepDiver1975) [reply Makefile:60]: symlink? Where?

File: pkg/assets/embed.yml (hunk 3 above)
Comment (DeepDiver1975) [reply pkg/assets/embed.yml]: We are working on the packaging and release scripting for the next phoenix release - yes.

File: Makefile (hunk 4 above)
Comment (tboerger) [reply Makefile:60]: This change is totally fine to reset the assets folder which is ignored via .gitignore.

File: pkg/assets/embed.yml (hunk 3 above)
Comment (tboerger) [reply pkg/assets/embed.yml]: Than revert it here and add the strip prefix to tar command :)

**Comment by tboerger** [comment]
This seem to work, but IMHO you should replace the phoenix release tarball with a proper tarball... No .DS_Store, no doubled phoenix folder...

**Comment by DeepDiver1975** [comment]
> This seem to work, but IMHO you should replace the phoenix release tarball with a proper tarball... No .DS_Store, no doubled phoenix folder...

taken care of right now: https://github.com/owncloud/phoenix/issues/2827

**Comment by LukasHirt** [comment]
> you should replace the phoenix release tarball with a proper tarball...

The release has a new tar ball now. Both issues should be fixed. No more .DS_Store and phoenix subfolder.

**Comment by tboerger** [comment]
@DeepDiver1975 update the pr according to that.

**Comment by DeepDiver1975** [comment]
> The release has a new tar ball now. Both issues should be fixed. No more .DS_Store and phoenix subfolder.

phoenix folder still there ad a  . root folder

Screenshot from 2020-01-09 11-56-31

**Comment by DeepDiver1975** [comment]
@tboerger @butonic here we go now ....

**Review by butonic — APPROVED** [review]
tarball is clean!

**Comment by butonic** [comment]
hm
```
➜  ocis-phoenix git:(feature/22) ✗ make clean
go clean -i ./...
rm -rf bin dist assets
➜  ocis-phoenix git:(feature/22) ✗ make generate build
mkdir assets/ && curl -slL -o- https://github.com/owncloud/phoenix/releases/download/v0.2.6/phoenix.tar.gz | tar xvzf - -C assets/
appinfo/
appinfo/info.xml
apps/
apps/files/
apps/files/2.files.chunk.js
apps/files/5.files.chunk.js
apps/files/6.files.chunk.js
apps/files/3.files.chunk.js
apps/files/7.files.chunk.js
apps/files/1.files.chunk.js
apps/files/files.bundle.js
apps/files/8.files.chunk.js
apps/files/9.files.chunk.js
apps/files/4.files.chunk.js
apps/pdf-viewer/
apps/pdf-viewer/pdf-viewer.bundle.js
apps/pdf-viewer/vendors~pdfjsWorker.pdf-viewer.chunk.js
apps/pdf-viewer/2.pdf-viewer.chunk.js
apps/pdf-viewer/3ab3a1f884f4022d6fad.worker.js
apps/pdf-viewer/3.pdf-viewer.chunk.js
apps/media-viewer/
apps/media-viewer/media-viewer.bundle.js
apps/draw-io/
apps/draw-io/draw-io.bundle.js
apps/markdown-editor/
apps/markdown-editor/markdown-editor.bundle.js.map
apps/markdown-editor/markdown-editor.bundle.js
CHANGELOG.md
config.json
core/
core/core.bundle.js
css/
css/core.css
img/
img/app.png
img/app.svg
index.html
LICENSE/
LICENSE/LICENSE
manifest.json
node_modules/
node_modules/requirejs/
node_modules/requirejs/require.js
oidc-callback.html
oidc-silent-redirect.html
README.md
themes/
themes/example/
themes/example/logo-icon.svg
themes/example.json
themes/owncloud.json
themes/owncloud/
themes/owncloud/favicon.jpg
.htaccess
go generate github.com/owncloud/ocis-phoenix/cmd/ocis-phoenix github.com/owncloud/ocis-phoenix/pkg/assets github.com/owncloud/ocis-phoenix/pkg/command github.com/owncloud/ocis-phoenix/pkg/config github.com/owncloud/ocis-phoenix/pkg/flagset github.com/owncloud/ocis-phoenix/pkg/metrics github.com/owncloud/ocis-phoenix/pkg/server/debug github.com/owncloud/ocis-phoenix/pkg/server/http github.com/owncloud/ocis-phoenix/pkg/service/v0 github.com/owncloud/ocis-phoenix/pkg/version
panic: No files found

goroutine 1 [running]:
main.main()
        /home/jfd/go/pkg/mod/github.com/!unno!ted/fileb0x@v1.1.4/main.go:101 +0x26c8
exit status 2
pkg/assets/assets.go:12: running "go": exit status 1
make: *** [Makefile:80: generate] Fehler 1
➜  ocis-phoenix git:(feature/22) ✗
```

**Pull request merged**